configure.ac

```diff
@@ -888,7 +900,12 @@ AC_CHECK_FUNCS([nanosleep], [],
     [
       AC_CHECK_LIB([posix4], [nanosleep],
         [nanosleep_needs_posix4="yes"],
-        [AC_MSG_ERROR([cannot find nanosleep])]
+        [
+          AC_CHECK_FUNCS([rpl_nanosleep],
+            [nanosleep_needs_gnulib="yes"],
```

Comment:
This seems to be unused. Do we need it?

Reply:
I think this is leftover from changes I patched in from kele's port, so it's not needed anymore.